Assets/Mirror/Components/NetworkTransform2k/NetworkTransformBase.cs

```diff
@@ -149,6 +165,26 @@ protected virtual void ApplySnapshot(NTSnapshot start, NTSnapshot goal, NTSnapsh
                 targetComponent.localScale = interpolateScale ? interpolated.scale : goal.scale;
         }
 
+        protected virtual bool CompareSnapshots(NTSnapshot currentSnapshot)
```

Comment:
Compare magnitutes, not square magnitutes as calculating the square root is more complex